tasks/get.yaml

```diff
@@ -156,25 +156,36 @@
             vaulted_text: "{{ sandbox_aws_secret_access_key }}"
           include_tasks: unvault.yml
 
-- name: Save vars in aws_sandbox_result dictionary
+- name: Save secret of aws_sandbox_result dictionary
   set_fact:
-    aws_sandbox_result:
-      sandbox_name: "{{ sandbox_name }}"
-      sandbox_account: "{{ sandbox_account }}"
-      sandbox_zone: "{{ sandbox_zone }}"
-      sandbox_hosted_zone_id: "{{ sandbox_hosted_zone_id }}"
-      sandbox_aws_access_key_id: "{{ sandbox_aws_access_key_id }}"
-      sandbox_aws_secret_access_key: "{{ sandbox_aws_secret_access_key }}"
-
+    aws_sandbox_secrets:
       # agnosticd
-      HostedZoneId: "{{ sandbox_hosted_zone_id }}"
-      subdomain_base_suffix: ".{{ sandbox_zone }}"
       aws_access_key_id: "{{ sandbox_aws_access_key_id }}"
       aws_secret_access_key: "{{ sandbox_aws_secret_access_key }}"
 
-- name: Inject vars into dynamic_job_vars
+- name: Inject secrets into dynamic_job_vars
   set_fact:
     dynamic_job_vars: >-
       {{ vars.dynamic_job_vars
       | default({})
-      | combine(aws_sandbox_result, recursive=True) }}
+      | combine(aws_sandbox_secrets, recursive=True) }}
+
+- name: Set sandbox for {{ anarchy_subject_name }}
+  anarchy_subject_update:
+    skip_update_processing: true
+    metadata:
+      labels:
+        sandboxed: true
+        sandbox: "{{ sandbox_name }}"
```

Comment:
A label cannot have a boolean value, but we wouldn't need a `sandboxed` label anyway, just search with a label selector without a value for `sandbox` will return everything with any value for this label.